crates/sui-graphql-client/src/lib.rs

```diff
@@ -115,7 +116,7 @@ const TESTNET_HOST: &str = "https://sui-testnet.mystenlabs.com/graphql";
 const DEVNET_HOST: &str = "https://sui-devnet.mystenlabs.com/graphql";
 const LOCAL_HOST: &str = "http://localhost:9125/graphql";
 static USER_AGENT: &str = concat!(env!("CARGO_PKG_NAME"), "/", env!("CARGO_PKG_VERSION"),);
-
+static MAX_PAGE_SIZE: OnceCell<i32> = OnceCell::const_new();
```

Comment:
Lets instead have this information live in the Client itself. That way its specific to the endpoint and not a globally enforced value. Imagine the case where you have two clients, each connected with a separate endpoint with different max page size settings.

Reply:
Went that route initially but did not think of the case of having two clients at different endpoints! Thanks for the fix.